app/src/main/java/com/spielpark/steve/bernieapp/fragments/NewsFragment.java

```diff
@@ -7,47 +7,135 @@
+import com.spielpark.steve.bernieapp.actMainPage;
+import com.spielpark.steve.bernieapp.misc.ImgTxtAdapter;
+import com.spielpark.steve.bernieapp.model.news.NewsArticle;
+import com.spielpark.steve.bernieapp.model.news.NewsManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Action1;
 
 /**
- * A placeholder fragment containing a simple view.
+ * A placeholder instance containing a simple view.
  */
 public class NewsFragment extends Fragment {
 
-    private static NewsFragment mIntstance;
+
+    private static NewsFragment instance;
+    @Bind(R.id.listNews) ListView list;
+    @Bind(R.id.progressBar) ProgressBar progressBar;
+    @Bind(R.id.txtSubHeader) TextView subHeader;
+    @Bind(R.id.txtHeader) TextView header;
+    ImgTxtAdapter adapter;
+    NewsArticle headerArticle;
+    private Subscription newsSubscription;
 
     public static NewsFragment getInstance() {
-        if (mIntstance == null) {
-            mIntstance = new NewsFragment();
-            return mIntstance;
+        if (instance == null) {
+            instance = new NewsFragment();
+            return instance;
         } else {
-            return mIntstance;
+            return instance;
         }
     }
 
     @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.frag_newsarticles, container, false);
+    }
+
+    @Override
     public void onViewCreated(final View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        final ListView newsList = (ListView) view.findViewById(R.id.listNews);
-        ((TextView) view.findViewById(R.id.txtSubHeader)).setMovementMethod(
-                new ScrollingMovementMethod());
-        new NewsTask(getActivity(), newsList, (ProgressBar) view.findViewById(R.id.progressBar),
-                (TextView) view.findViewById(R.id.txtSubHeader),
-                (TextView) view.findViewById(R.id.txtHeader)).execute();
+        ButterKnife.bind(this, view);
+        subHeader.setMovementMethod(new ScrollingMovementMethod());
+        adapter = new ImgTxtAdapter(view.getContext(), R.layout.list_news_item, new ArrayList());
+        list.setAdapter(adapter);
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.frag_newsarticles, container, false);
+    public void onResume() {
+        super.onResume();
+        newsSubscription = NewsManager.get().getNews().observeOn(AndroidSchedulers.mainThread()).subscribe(new Action1<Pair<NewsArticle, List<NewsArticle>>>() {
```

Comment:
Would this be easier if you created a observer field that you could pass to `subscribe(myObserver)` ?